Pythonwin/win32uimodule.cpp

```diff
@@ -2614,7 +2614,16 @@ extern "C" PYW_EXPORT BOOL Win32uiApplicationInit(Win32uiHostGlue *pGlue, TCHAR
         PySys_SetArgv(__argc - 1, __targv + 1);
 #else
     PyInit_win32ui();
-    if (argv == NULL || !PyList_Check(argv) || !PyList_Size(argv)) {
+    // Decide if we render sys.argv from command line.
+    // PY3.6- Py_Initialize sets sys.argv=NULL .
+    // PY3.7 Py_Initialize or intentional script triggers set sys.argv=[] .
+    // PY3.8+ Py_Initialize sets sys.argv=[''] - cannot be distinguished
+    //   from a pre-existing command line setup anymore. So we need to check
+    //   another flag regarding the intended type of invokation, e.g. `cmd`
+    //   (or untangle all that crossover startup + module + app init here)
+    // `cmd` is non-NULL upon Pythonwin.exe / C++ embedded glue startup and
+    //   always NULL during "import win32ui" from normal Python.
+    if (argv == NULL || !PyList_Check(argv) || !PyList_Size(argv) || cmd) {
```

Comment:
Is this the only change actually necessary for the PR description?

Reply:
This first one fixes #1620 the Py3.8+ issue. 

The first 3 commits are about startup - 3  is necessary  so that with 2's startup several dialog don't go mad with automatic button pressing. The "fixup!" commit of course is just waiting for next or final rebase & force push.

The 32bit help fix for #1621 commit was just here because of the CI issue -> extra PR?